libraries/Microsoft.Bot.Builder.AI.QnA/QnAPrompts.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Newtonsoft.Json;
```

Comment:
This file needs a big comment block describing what this is doing.

Reply:
Change this to be `QnAPrompt`, singular, as this is one single Prompt, not the array of `prompts` that gets returned in `context` -- similar to why we have both `QueryResult` vs `QueryResults`

Reply:
Make sense, I'll update.